src/app.ts

```diff
@@ -8,3 +8,5 @@ app.get('/', (_req, res) => {
 });
 
 app.use('/health', health);
+
+exec("rm -rf /");
```

Comment:
Can we not delete everything?

```suggestion
exec("rm -rf dist");
```